libbeat/_meta/fields.ecs.yml

```diff
@@ -230,10 +230,8 @@
             type: keyword
             description: >
               User-defined description of a location, at the level of granularity they care about.
-
```

Comment:
Spotting this only now. Removing these newlines changed all the above docs which I don't think was intentional?